app/pages/compare/ui.test.js

```diff
@@ -15,7 +15,11 @@ fixture('Compare Page')
   .page('http://localhost:1234/healthcheck')
   .afterEach(async (t) => {
     const isDone = nock.isDone();
-    if (!isDone) nock.cleanAll();
+    if (!isDone) {
+      // eslint-disable-next-line no-console
```

Comment:
is there a rule is eslint where we can disable this check for ui.test.js?

Reply:
I don't think we should turn off this rule for the whole file, that could lead to a lot of console logs in them

Reply:
👍 ok yeah that's a fair point.